grails-app/controllers/au/org/ala/profile/api/ApiController.groovy

```diff
@@ -510,6 +510,136 @@ class ApiController extends BaseController {
         }
     }
 
+    @Path("/api/opus/{opusId}/profile/{profileId}/image/{imageId}")
+    @Operation(
+            summary = "Get image associated with a profile",
+            operationId = "/api/opus/{opusId}/profile/{profileId}/image/{imageId}",
+            method = "GET",
+            responses = [
+                    @ApiResponse(
+                            responseCode = "200",
+                            content = @Content(
+                                    mediaType = "image/*",
+                                    schema = @Schema(type = "String", format = "binary")
+                            ),
+                            headers = [
+                                    @Header(name = 'Access-Control-Allow-Headers', description = "CORS header", schema = @Schema(type = "String")),
+                                    @Header(name = 'Access-Control-Allow-Methods', description = "CORS header", schema = @Schema(type = "String")),
+                                    @Header(name = 'Access-Control-Allow-Origin', description = "CORS header", schema = @Schema(type = "String"))
+                            ]
+                    ),
+                    @ApiResponse(responseCode = "400",
+                            description = "opusId and profileId are required parameters"),
+                    @ApiResponse(responseCode = "403",
+                            description = "You do not have the necessary permissions to perform this action."),
+                    @ApiResponse(responseCode = "405",
+                            description = "An unexpected error has occurred while processing your request."),
+                    @ApiResponse(responseCode = "404",
+                            description = "Opus or profile not found"),
+                    @ApiResponse(responseCode = "500",
+                            description = "An unexpected error has occurred while processing your request.")
+            ],
+            parameters = [
+                    @Parameter(name = "opusId",
```

Comment:
`in` parameter missing/removed. It is required. Read more on how to write api documentation from https://swagger.io/docs/specification/2-0/basic-structure/